rpc/tendermint/test/websocket_client_test.go

```diff
@@ -260,29 +260,32 @@ func TestSubscribe(t *testing.T) {
 	var subId string
 	subscribe(t, wsc, txs.EventStringNewBlock())
 
-	timeout := time.NewTimer(timeoutSeconds * time.Second)
+	// timeout to check subscription process is live
+	timeout := time.After(timeoutSeconds * time.Second)
 Subscribe:
 	for {
 		select {
-		case <-timeout.C:
+		case <-timeout:
 			t.Fatal("Timed out waiting for subscription result")
 
 		case bs := <-wsc.ResultsCh:
 			resultSubscribe, ok := readResult(t, bs).(*core_types.ResultSubscribe)
 			if ok {
 				assert.Equal(t, txs.EventStringNewBlock(), resultSubscribe.Event)
 				subId = resultSubscribe.SubscriptionId
+				fmt.Printf("\x1b[30m\x1b[46mMARMOT!: subid %s\x1b[0m\n", subId)
```

Comment:
Haha. Dammit those special ops marmots can be hell to extract from the field once they've gone to ground.